samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -144,6 +129,78 @@ class DailyCallManager {
     }
   }
 
+  /**
+   * Handles participant-joined, participant-updated, and participant-left events:
+   * - Updates the participant count
+   * - Adds or removes video and audio tracks for the participant based on the event and track state
+   * @param {Object} event - The participant-* event object.
+   */
+  handleNewParticipantState(event) {
+    // Update the participant count
+    this.updateAndDisplayParticipantCount();
+
+    // Get the audio and video tracks and participant ID from the participant-* events
+    const tracks = event.participant.tracks;
+    const participant = event.participant;
+    const participantId = participant.session_id;
+
+    if (event.action === 'participant-left') {
+      if (participant) {
+        // Remove the video container if it exists
+        const videoContainer = document.getElementById(
+          `video-container-${participantId}`
+        );
+        if (videoContainer) {
+          this.destroyTrack('video', participantId);
+          videoContainer.remove();
+        }
+
+        // Remove the audio element if it exists
+        const audioElement = document.getElementById(`audio-${participantId}`);
+        if (audioElement) {
+          this.destroyTrack('audio', participantId);
+        }
+      }
+    } else {
+      for (const trackType in tracks) {
+        if (tracks[trackType].state === 'playable') {
+          if (trackType === 'video') {
+            this.displayVideo(tracks[trackType], participantId);
+          } else if (trackType === 'audio') {
+            this.playAudio(tracks[trackType], participant);
+          } else {
+            // Actions for screenVideo and screenAudio tracks if needed.
+          }
+        } else if (
+          (tracks[trackType].state === 'off') |
```

Comment:
i think you'd need to do `(tracks[trackType].state === 'off') || (tracks[trackType].state === 'interrupted') || (tracks[trackType].state === 'blocked')`.

it might be better to turn this whole `if/else` statement into a `switch/case` one instead.

Reply:
Whoops, yes.

Yeah, that might make readability easier.